github/ci/prow-deploy/kustom/base/manifests/local/prow-phased-deployment.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: phased
+  labels:
+    app: phased
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: prow-phased
+  template:
+    metadata:
+      labels:
+        app: prow-phased
+    spec:
+      terminationGracePeriodSeconds: 180
+      containers:
+        - name: phased
+          image: quay.io/kubevirtci/phased:v20240108-87253b59
+          imagePullPolicy: IfNotPresent
+          args:
+            - --github-token-path=/etc/github/oauth
+            - --github-endpoint=http://ghproxy
```

Comment:
seems we can remove this line because it is duplicate and the next line will be the actual value
(copy paste from rehearse plugin) but not critical, just cosmetics